### ISSUE TITLE
[Question] What's the difference between `in_txt` and `out_txt` fields in `tlog.log` file? 

### ISSUE THREAD

**Issue by haiwu** [Scribery/tlog#359]
By default, it seems `tlog` would not log user input in `in_txt` field, per default config for `/etc/tlog/tlog-rec-session.conf`. Thus user inputs would show up in `out_txt` field instead. 

We could toggle config in tlog-rec-session.conf file to log user intput in `in_txt` field via:
"log": {
  "input": true,
...
}

My question is: what the point of not logging user input in `in_txt` field by default? If for security concerns, user inputs still got logged in `out_txt` field in clear-text anyway. 

Please clarify.

**Comment by haiwu** [comment]
Just saw https://github.com/Scribery/tlog/issues/208. I think I kind of get it now. Password by default is echo disabled.

**Comment by justin-stephenson** [comment]
> Just saw #208. I think I kind of get it now. Password by default is echo disabled.

That's correct, the intention is to prevent security-sensitive information from being included in recorded data by default.
